### PR TITLE
Introducing Nhost Guru on Gurubase.io

### PR DESCRIPTION
**Pull request by kursataktas**
Hello team,

I'm the maintainer of [Anteon](https://github.com/getanteon/anteon). We have created Gurubase.io with the mission of building a centralized, open-source tool-focused knowledge base. Essentially, each "guru" is equipped with custom knowledge to answer user questions based on collected data related to that tool.

I wanted to update you that I've manually added the [Nhost Guru](https://gurubase.io/g/nhost) to Gurubase. Nhost Guru uses the data from this repo and data from the [docs](https://docs.nhost.io) to answer questions by leveraging the LLM.

In this PR, I showcased the "Nhost Guru", which highlights that Nhost now has an AI assistant available to help users with their questions. Please let me know your thoughts on this contribution.

Additionally, if you want me to disable Nhost Guru in Gurubase, just let me know that's totally fine.

### CODE REVIEW

**Comment by kursataktas** [comment]
Hi everyone,

Have you had a chance to check this out?

Additionally, I’d like to update you on the release of the Maintainer Panel feature on Gurubase. With this panel, you can add, remove, or update data sources, as well as change the logo and more. You can find the details [here](https://github.com/getanteon/gurubase?tab=readme-ov-file#how-to-claim-a-guru). 

In the near future, I’m planning to include analytics insights in this panel, such as the number of questions asked, the most frequently asked ones, and more. I’ll be sure to update this thread once it’s available. However, in case I miss it, I highly recommend joining our [Discord channel](https://discord.gg/9CMRSQPqx6) to stay updated.

If you’d prefer that I don’t update this thread anymore, please let me know.

**Comment by dbarrosop** [comment]
/cc @nunopato 

thanks for this, this looks very cool. Just for us to understand this a bit better I have a few questions if you don't mind:

1. Is this part of a soon-to-be-monetized project?
2. Do you have plans for integrations? I.e. imagine our dashboard having an embedded chat to gurubase or in our docs (we have something similar through mintlify today, which uses some external service I can't recall right now so curious if you have plans for a similar service).

Congrats on the work in any case, this is very cool. And, please, keep updating while we figure this out. Worst case we will add this as a "third party/unofficial" source of information.

**Comment by kursataktas** [comment]
Thanks for the review @dbarrosop 

> Is this part of a soon-to-be-monetized project?

Gurubase is maintained by the Anteon team. Anteon is our commercial product, whereas Gurubase is a side project. But I can’t make any promises for the future. Even if it includes pricing someday, I don’t think charging non-commercial open-source projects is a valid business strategy. 

> Do you have plans for integrations? I.e. imagine our dashboard having an embedded chat to gurubase or in our docs (we have something similar through mintlify today, which uses some external service I can't recall right now so curious if you have plans for a similar service).

This is the most requested feature, and I think we will develop it eventually. However, we haven't finalized an ETA yet. I can update this thread once it's developed and released. In the meantime, I highly recommend joining our [Discord channel](https://discord.gg/9CMRSQPqx6) and following our [Twitter account](https://x.com/gurubaseio) to stay updated on new features.

File: README.md [hunk 1]
```diff
@@ -15,6 +15,8 @@
 <a href="https://twitter.com/nhost">Twitter</a>
 <span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
 <a href="https://nhost.io/discord">Discord</a>
+<span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
+<a href="https://gurubase.io/g/nhost">Ask Nhost Guru</a>
```
Comment (dbarrosop) [inline README.md:19]:
```suggestion
<a href="https://gurubase.io/g/nhost">Ask Nhost Guru (third party, unofficial)</a>
```

**Comment by dbarrosop** [comment]
Ok, I have added a note that this is coming from a third party/unofficial but other than that merging now. I will go now and claim the guru. Will start using it a bit and we may remove the "third party" note a bit down the road.

Thanks again for this.

**Review by dbarrosop — APPROVED** [review]

**Comment by kursataktas** [comment]
Hello @dbarrosop,

We recently released the embeddable "Ask AI" widget, which allows you to integrate your Guru’s Q&A capability into websites, documentation, and more.

The installation process is straightforward, as explained in [this GitHub repository](https://github.com/getanteon/gurubase-widget).

You can see it in action on the [Anteon](https://getanteon.com/) and [NetAlertX](https://netalertx.com/) websites.

**Comment by dbarrosop** [comment]
Apologies for the late response, thanks a lot for the heads up, this looks amazing. Awesome work!

**Pull request merged**